src/components/Partners.astro

```diff
@@ -54,6 +54,10 @@ import HeadingFlanked from "./HeadingFlanked.astro";
 		max-width: 90vw;
 	}
 
+	.partners-blurb {
+		margin: 0 auto;
+	}
```

Comment:
Oh man if I'd known this needed CSS fixing I would have offered to do this for you 😄 thank you!